style_guide.md

```diff
@@ -0,0 +1,18 @@
+# Style Guide
+
+Please follow the guidelines stated here when contributing to this repository.
+
+## Spacing
+
+- Always use tabs. Spaces suck.
+- Avoid trailing spaces at the end of lines.
+- Avoid trailing empty lines at the end of files.
+- Surround operators with a single space (except for when using `=` in parameter lists).
+- Leave two empty lines between functions.
+- Optionally leave one empty line between logical blocks of code in the same function.
+
+## Naming
```

Comment:
Should also mention our preferred prefixes and names for files/directories

Reply:
Such as? Stuff like "name you cog class file `my_feature_cog.py`" has been changed. Check out #28